### ISSUE TITLE
Add CODEOWNERS file for automatic PR review requests

### ISSUE THREAD

**Issue by wesley-dean-gsa** [GSA/usagov-benefits-eligibility#264]
As a contributor to the BEARS repository, so that my Pull Requests (PRs) are automatically assigned a reviewer, I would like automation added to make sure new PRs are assigned a reviewer upon submission based on what's being updated.

GitHub allows a `CODEOWNERS` file that maps files included in a PR's commits (e.g., `*.js` could go to developers, anything in `/content/` could go to content editors, etc.).  [Additional documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) is available online.

Sample:

```
*.js *.scss *.vue @bears-engineers !@wesley-dean-gsa
*.md ! /content/ @bears-engineers !@wesley-dean-gsa
/content/ @bears-content !@wesley-dean-gsa !@jonadecker
```

**Comment by wesley-dean-gsa** [comment]
we'll put the file at `/.github/CODEOWNERS`

**Comment by wesley-dean-gsa** [comment]
https://github.com/GSA/usagov-benefits-eligibility/settings/branch_protection_rules/24508234

^ sample branch protection rule (applies to `main-draft` ; will need to change to apply to `main` to make it live)

**Comment by wesley-dean-gsa** [comment]
resolves #181
